test/reader/StorageReadable.t.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We decided to settle on LGPL, the same as for the majority of the files in this repo.
Would you mind updating the licensing of this file?

```suggestion
// SPDX-License-Identifier: LGPL-3.0-or-later
```